frontend/common/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+import { usePaginationVars } from ".";
+import { Skill } from "../../api/generated";
+import { fakeSkills } from "../../fakeData";
+import Pagination, { PaginationProps } from "./Pagination";
+
+export default {
+  component: Pagination,
+  title: "Components/Pagination",
+  args: {
+    handlePageChange: action("Change page"),
+  },
+} as Meta;
+
+const TemplatePagination: Story<PaginationProps> = (args) => {
+  return <Pagination {...args} />;
+};
+
+export const NoDots = TemplatePagination.bind({});
+export const RightDots = TemplatePagination.bind({});
+export const LeftDots = TemplatePagination.bind({});
+export const BothDots = TemplatePagination.bind({});
+
+NoDots.args = {
```

Comment:
What about having a Default (maybe not even exported)

Then just merge the variations (less noisy about what's unique in each)

```js
Default.args = {
  totalCount: 100,
  siblingCount: 1,
  currentPage: 1,
  pageSize: 10,
};

NoDots.args = {
  ...Default.args,
  totalCount: 50,
};

RightDots.args = {
  ...Default.args,
};

LeftDots.args = {
  ...Default.args,
  currentPage: 9,
};

BothDots.args = {
  ...Default.args,
  currentPage: 5,
};
```

Reply:
Excellent point